php/docker/templates/nginx-vhost-conf.d/80-index.conf

```
@@ -1,6 +1,7 @@
 # vim: set ft=nginx:
 
 {{- $pageCacheEnabled := default "true" .Env.STACK_PAGE_CACHE_ENABLED }}
+{{- $enablePhpWhitelisting := default "true" .Env.STACK_WP_PHP_WHITELIST_ENABLED }}
```

Actually something like this would be more suggestive. Having `PHP_WHITELIST` disabled, would lead you to think that all php paths would be blocked by default, but it would mean the opposite.
```suggestion
{{- $enablePhpWhitelisting := default "true" .Env.STACK_BLOCK_NONSTANDARD_PHP_PATHS }}
```